src/icons/accessible-svg/AccessibleSVG.tsx

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+import omit from 'lodash/omit';
+import uniqueId from 'lodash/uniqueId';
+import { SVGProps } from '../../components/accessible-svg/AccessibleSVG';
+
+export interface AccessibleSVGIconProps extends SVGProps {
+    'aria-labelledby'?: string;
+    focusable?: boolean | 'false' | 'true' | 'auto' | undefined;
+    opacity?: number;
+}
+
+class AccessibleSVG extends React.Component<AccessibleSVGIconProps> {
+    id: string = uniqueId('icon');
+
+    render() {
+        const { children, title, ...rest } = this.props;
+        const titleID = `${this.id}-title`;
```

Comment:
curious, do we need the `id: string`? if we were to put `uniqueId('icon')` in line 17, could we make this a functional component?

I don't think the change is needed now, since this is mostly a conversion, but wasn't sure if the class attribute was needed.

Reply:
Good idea! This seems like a minor change, so I'll see if I can include it in this PR

Reply:
well you don't want to generate a new unique id every render, so you'll have to keep it in state

Reply:
That makes sense. Let's keep this component as is for now. I'll add a note about this in the migration Box Note.